### PR TITLE
README.md and CONTRIBUTING.md revision

### PR DESCRIPTION
**Pull request by marcospb19**
Aditionally wrote a brief tutorial on CONTRIBUTING.md on how to implement a tool from scratch.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,98 +1,95 @@
...
 
-## Styleguides
+Now you can start editing files inside of the new directory, here's a list.
+- Replace _"template"_ occurrences in [`build.rs`](.template/build.rs) by the TOOL_NAME.
+- Make changes in [`Cargo.toml`](.template/Cargo.toml) indicated fields.
+- Configure argument parsing with `clap` at [`cli.rs`](.template/src/cli.rs), here are some rules:
+  - Every `Arg` needs `.help()`, `.long()` and `.short()` options.
+  - Short options respect the [POSIX standard](https://pubs.opengroup.org/onlinepubs/9699919799/idx/utilities.html).
+  - Long options need to match the widely common GNU's names.
+- Every help and error message should start with upper case letter and end with a dot.
```
Comment (GrayJack) [inline CONTRIBUTING.md]: Only help messages, there is no styleguide for the error messages... yet. I want to, but right now every tool uses a little bit different (but very similar) error message. I would like to think and discuss more before deciding and just put in the guidelines after all tools are compliant with the new styleguide for error messages

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,98 +1,95 @@
...
+```
 
-## Styleguides
+Now you can start editing files inside of the new directory, here's a list.
+- Replace _"template"_ occurrences in [`build.rs`](.template/build.rs) by the TOOL_NAME.
+- Make changes in [`Cargo.toml`](.template/Cargo.toml) indicated fields.
+- Configure argument parsing with `clap` at [`cli.rs`](.template/src/cli.rs), here are some rules:
+  - Every `Arg` needs `.help()`, `.long()` and `.short()` options.
+  - Short options respect the [POSIX standard](https://pubs.opengroup.org/onlinepubs/9699919799/idx/utilities.html).
+  - Long options need to match the widely common GNU's names.
```
Comment (GrayJack) [inline CONTRIBUTING.md]: What about if the feature are a extension? It's not defined in the standard, then what should the person implementing the tool suppose to do?

I think it should look in the extensions of the other coreutils implementation manual pages and see if what they are doing were done before and use that for `short` and `long`, and if only one of the two is defined, then feel free to create a new one.
If it's something new, feel free to create a `short` and a `long`.

Long options should prefer match what is called on the majority of the other implementation of the tools, and if makes sense to change, add a visible alias with the common long name.
It's also nice to have alias for other long names, if there is more than one, for compatibility reasons.

File: README.md [hunk 3]
```diff
@@ -3,55 +3,52 @@
...
-This project has no intent to be 100% compatible with GNU's coreutils, like [Uutils' coreutils](https://github.com/uutils/coreutils), but if happens to be, that's ok too.
+An in-progress safe implementation of `coreutils` aiming for a minimal and yet complete set of utilities. This project aims at the _POSIX_ specification basic requirements, plus common and useful features present in other implementations of the utility.
 
-## Contributing
-Check the [CONTRIBUTING.md](./CONTRIBUTING.md) file for the guidelines to contribute to the project, including issue reports, git commits messages, etc.
+This project has no intent to be compatible with _GNU's coreutils_, like [Uutils' coreutils](https://github.com/uutils/coreutils).
 
 ## Minimum Rust Version Policy
-This project's minimum supported `rustc` version (MSRV) is 1.40.0.
+This project's minimum supported `rustc` version (_MSRV_) is _1.40.0_.
```
Comment (GrayJack) [inline README.md]: 1.43.0 is the MSRV

File: README.md [hunk 4]
```diff
@@ -3,55 +3,52 @@
...
+This will not be conservative until we get to a _1.0_ version. So it can be changed at any point in time.
 
 <!-- In general, this project will try to be conservative with respect to the minimum supported version of Rust, but in case of safety reasons it may bump at any time [e.g. `MaybeUninit` stabilization on 1.36.0 fixing huge problems with `std::mem::uninitialized()`] or improvements that affect positively conditional compilation (we use it a lot). -->
 
 ## Compilation tests guarantees
-The compilations is tested for Rust Tier 1 and Tier 2 x86_64 Unix/Unix-like platforms, with exception of redox target, with Github Actions.
+Compilation is tested for Rust Tier 1 and Tier 2 _x86_64 Unix/Unix-like_ platforms (except _Redox_), with _CI_.
 
-The Tier1 platforms are tested on stable, beta and nightly Rust, while Tier2 and Tier3 platforms are tested on stable only, since they are no guarantees to be available in beta and nightly Rust.
+All platforms are tested on _stable_ _Rust_, and Tier 1 platforms are also tested on _beta_ and _nightly_ _Rust_ (Tier 2 and 3 only guarantee _stable_ full capacity).
```
Comment (GrayJack) [inline README.md]: All platforms are tested on MSRV and stable Rust

**Review by GrayJack — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (marcospb19) [reply README.md]: ok

File: README.md (hunk 3 above)
Comment (marcospb19) [reply README.md]: So, it should be replaced by

"This project's minimum supported `rustc` version (_MSRV_) is _1.43.0_."

?

File: CONTRIBUTING.md (hunk 1 above)
Comment (marcospb19) [reply CONTRIBUTING.md]: Ok, I'll remove the "and error message" part.

File: CONTRIBUTING.md (hunk 2 above)
Comment (marcospb19) [reply CONTRIBUTING.md]: I rewrote some, commiting.

**Comment by marcospb19** [comment]
I made several changes, can you review again? @GrayJack.

**Pull request merged**